### ISSUE TITLE
Suggest a better default Sk.read() default implementation

### ISSUE THREAD

**Issue by csev** [skulpt/skulpt#47]
It would seem to me that we should provide a decent SK.read() for the in-browser case.   It seems less than ideal to expect everyone who wants to use this in the browser to find and paste the code into their HTML pages:

```
function builtinRead(x)
{
    if (Sk.builtinFiles === undefined || Sk.builtinFiles["files"][x] === undefined)
        throw "File not found: '" + x + "'";
    return Sk.builtinFiles["files"][x];
}
```

And

```
Sk.configure({output:outf, read: builtinRead});
```

Just to get import statements working.

My recommendation is to put this code in the Sk.read() implementation in env.js - but only active is Sk.inBrowser is true.

The alternative (less attractive) is to do a bit more logic around the try/catch block in the Sk.importSearchPathForName implementation in import.js.

My preference is in env.js.  If the user wants a more sophisticated Sk.read() implementation, they can just override the default in the Sk.configure() call.

**Comment by albertjan** [comment]
I think this is a good idea, I'm working on a fix to `import` that'll make `from bla import something` work for python packages.

**Comment by bnmnetp** [comment]
@albertjan, I pushed a branch called cherry-picks that has such a fix in it.  It was mixed in with some other stuff for @eah13, so I did not make a PR, but if @eah13 has not incorporated that change into another PR you are welcome to look at it and use it.

@csv, I have long been an advocate of separating the builtinRead required for import from a more general read for file I/O simulation.   We certainly don't need to wait, but this is another reason behind my proposal this morning to look at a new more user friendly API for skulpt. See #446   Would be interested in your thoughts.
